server/enterprise.go

```diff
@@ -28,7 +29,7 @@ func (s *Server) triggerEnterpriseTests(pr *model.PullRequest) {
 		return
 	}
 
-	isBaseBranchReleaseBranch, err := regexp.MatchString(`$release-*`, triggerInfo.BaseBranch)
+	isBaseBranchReleaseBranch, err := regexp.MatchString(`^release-*`, triggerInfo.BaseBranch)
```

Comment:
Woops

Reply:
yes! I really need to get this PR deployed.